AppCenter/AppCenter/MSACLogger.h

```diff
@@ -41,4 +42,6 @@
           function:(const char *)function
               line:(uint)line;
 
++ (MSACLogHandler)getCurrentLogHandler;
```

Comment:
You can remove the "get" in the name because it's obvious that it returns a loghandler.

Reply:
Usually getters don't start with "get" unless they don't return explicitly what they are supposed to return.

Reply:
Fixed. Also moved the definition to `MSACLoggerInternal.h` and named `logHandler` for the sake of consistency to the setter `setLogHandler`.